docs/source/rest-api.rst

```diff
@@ -193,6 +193,89 @@ Response Structure
 
 
 
+.. _mlflowMlflowServicedeleteExperiment:
+
+Mlflow Experiments Delete
+=========================
+
+
++-------------------------------------------+-------------+
+|                 Endpoint                  | HTTP Method |
++===========================================+=============+
+| ``2.0/preview/mlflow/experiments/delete`` | ``POST``    |
++-------------------------------------------+-------------+
+
+Mark an experiment and associated runs, params, metrics, ... etc for deletion.
+If the experiment uses FileStore, artifacts associated with experiment are also deleted.
+
+
+
+
+.. _mlflowDeleteExperiment:
+
+Request Structure
+-----------------
+
+
+
+
+
+
++---------------+-----------+---------------------------------+
+|  Field Name   |   Type    |           Description           |
++===============+===========+=================================+
+| experiment_id | ``INT64`` | Identifier to get an experiment |
+|               |           | This field is required.         |
```

Comment:
Identifier to get an experiment.

Reply:
I changed this description to be the same as the others on this page.